maintainers/maintainer-list.nix

```diff
@@ -5311,6 +5311,11 @@
     github = "viric";
     name = "Lluís Batlle i Rossell";
   };
+  virusdave = {
+    email = "dave.nicponski@gmail.com";
```

Comment:
Can't say i'm happy about this :(

Reply:
Being a maintainer or exposing this email?
You can use the noreply github email if desired.

Reply:
the email, although i guess i've already exposed that in prior public github commits, so i guess this is only marginal damage at most :)